src/main/java/com/emc/object/s3/S3ObjectMetadata.java

```diff
@@ -285,6 +311,14 @@ public String getDecodedUserMetadata(String name) {
         return RestUtil.urlDecode(getUserMetadata(name));
     }
 
+    public ObjectLockLegalHold getObjectLockLegalHold() {
```

Comment:
these new properties need setters

... also, can you set object lock with a metadata update call? (PUT+copy to self, replacing metadata)